ceexon/the-Questioner
# Ft ui admin add meetup 162965889

# Admin Profile/Dashboard 

**What this PR entails?**
UI template for the adminusers profile page

**Description of tasks completed**
- Html for admin to view their profile and meetup stats
- Css for admin profile + responsivenes :smile: 
- JAdmin add meetup 

**How to manually test it**
- Ensure you have internet
- Open your browser  
- click on this link 
 ```
       https://kburudi.github.io/the-Questioner/UI/templates/admin-page.html
```
- You will access the page

 **Related PT story**
#162965889
![admin-page-look](https://user-images.githubusercontent.com/45054246/50739447-91a7e200-11f1-11e9-8c8b-c3cff05cdf5f.png)
![admin-resp-bottom](https://user-images.githubusercontent.com/45054246/50739448-91a7e200-11f1-11e9-97de-9105dfbab3bd.png)
![admin-resp-top](https://user-images.githubusercontent.com/45054246/50739449-92407880-11f1-11e9-834c-69dcf26ed26b.png)
